docs/config.rst

```diff
@@ -421,11 +422,15 @@ Local
     :param fileext: The file extension to use (e.g. ``.txt``). Contained in the
         href, so if you change the file extension after a sync, this will
         trigger a re-download of everything (but *should* not cause data-loss
-        of any kind).
+        of any kind). To be compatible with the ``vset to the empty string`` format you have
```

Comment:
I think there's something wrong in this sentence.

Reply:
Fixed, thanks for noticing this "paste and copy" error.